frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1509,491 +1779,248 @@ function ReaderFooter:addToMainMenu(menu_items)
-                                    return self.settings.toc_markers_width == 2
-                                end,
-                                callback = function()
-                                    self.settings.toc_markers_width = 2  -- unscaled_size_check: ignore
-                                    self:setTocMarkers()
-                                    self:refreshFooter(true)
-                                end,
-                            },
-                            {
-                                text = _("Thick"),
-                                checked_func = function()
-                                    return self.settings.toc_markers_width == 3
-                                end,
-                                callback = function()
-                                    self.settings.toc_markers_width = 3  -- unscaled_size_check: ignore
-                                    self:setTocMarkers()
-                                    self:refreshFooter(true)
-                                end
-                            },
-                        },
                     },
                     {
-                        text = _("Show initial position marker"),
+                        text = _("Right"),
                         checked_func = function()
-                            return self.settings.initial_marker == true
+                            return self.settings.align == "right"
                         end,
                         callback = function()
-                            self.settings.initial_marker = not self.settings.initial_marker
-                            self.progress_bar.initial_pos_marker = self.settings.initial_marker
+                            self.settings.align = "right"
                             self:refreshFooter(true)
-                        end
+                        end,
                     },
-                },
+                }
             },
             {
                 text_func = function()
-                    local text = _("static margins (10)")
-                    local cur_width = self.settings.progress_margin_width
-                    if cur_width == 0 then
-                        text = _("no margins (0)")
-                    elseif cur_width == Screen:scaleBySize(material_pixels) then
-                        text = T(_("static margins (%1)"), material_pixels)
-                    end
-                    if self.settings.progress_margin and not self.ui.document.info.has_pages then
-                        text = T(_("same as book margins (%1)"), self.book_margins_footer_width)
-                    end
-                    return T(_("Margins: %1"), text)
-                end,
-                enabled_func = function()
-                    return not self.settings.disable_progress_bar
+                    return T(_("Items mini-bar height: %1"), self.settings.container_height)
```

Comment:
Container is significantly better. "Items mini-bar" is a rather convoluted paraphrase of status bar that doesn't add anything.

Reply:
I know, I agree it isn't pretty. the point is, a normal Joe won't know what that 'container' means. It removes the presumption of knowing code-things related.

Reply:
How about simply "total height"?

Reply:
I suppose that is better, what about `items height`? Anyway, in acceptance of my previous mistake, all `width`s are back. I suppose we are ready for the final final check.

Reply:
I feel like the items don't include the border but it could work. @poire-z ?

Reply:
We don't need to be that precise, whether it includes or not the border - after all, there is not unit :)
I'd use "Status bar height" if you don't want to use "container".  But I feel we don't need to use "Items" here, "Items height", it's not individual items, and we may have items smaller than that, ie. not line-height-tall symbols.

Reply:
Right, that's basically what I mean.